crane-cloud/multi-cluster
# Ft integrate metrics pipeline

### What does this PR do?

- It integrates CPU, Memory & Disk metrics
- Restructures codebase to collect all metrics from one `metrics.py` file
- Deletes redundant files
- 

### How can it be tested

- Pull branch locally, run the central server and peer apps from the compose script
- ssh into one of the peer apps & run client.py it should return CPU, memory and disk metrics
